gnomad/assessment/sanity_checks.py

```diff
@@ -384,7 +393,7 @@ def compare_subset_freqs(
     :param verbose: If True, show top values of annotations being checked, including checks that pass; if False, show only top values of annotations that fail checks.
     :param show_percent_sites: If True, show the percentage and count of overall sites that fail; if False, only show the number of sites that fail.
     :param delimiter: String to use as delimiter when making group label combinations. Default is "-".
-    :param metric_first_label: If True, metric precedes label subset, e.g. AC-non_v2-XY. If False, subset precedes metric, non_v2-AC-XY. Default is True.
+    :param metric_first_field: If True, metric precedes subset, e.g. AC-non_v2-. If False, subset precedes metric, non_v2-AC-XY. Default is True.
```

Comment:
Is this supposed to have an XY?

```suggestion
    :param metric_first_field: If True, metric precedes subset, e.g. AC-non_v2-XY. If False, subset precedes metric, non_v2-AC-XY. Default is True.
```

Reply:
Ah yes originally! I definitely got sidetracked here and forgot to go back.  I was trying to give an example for the types of comparisons the module does with the addition of the XY but I removed the a similar label in `make_group_sum_expr_dict`, which I think reduces confusion. It read 
```
:param metric_first_field: If True, metric precedes subset in the Table's fields, e.g. AC-hgdp. If False, subset precedes metric, hgdp-AC. Default is True.
``` 
Do you think removing the last -XY would reduce confusion here as well? I couldnt decide if the exta label. -XY, was helpful.

Reply:
yeah, maybe just remove both, I think that gets the point across